subgraph/package.json

```diff
@@ -17,6 +17,7 @@
     "create-local": "graph create --node http://localhost:8020/ ",
     "remove-local": "graph remove --node http://localhost:8020/ ",
     "deploy-local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001",
+    "introspect": "node bin/introspect.js",
```

Comment:
Actually can we make this run before _any_ `yarn build`?

Reply:
will add rn